server/protocols/webadmin-integration-test/webadmin-integration-test-common/src/test/java/org/apache/james/webadmin/integration/WebAdminServerIntegrationTest.java

```diff
@@ -58,13 +56,12 @@
 import org.eclipse.jetty.http.HttpStatus;
 import org.junit.After;
 import org.junit.Before;
-import org.junit.Rule;
 import org.junit.Test;
 
 import io.restassured.RestAssured;
 import io.restassured.http.ContentType;
 
-public class WebAdminServerIntegrationTest {
+public abstract class WebAdminServerIntegrationTest {
```

Comment:
Can memory pass all of this contract?

I suggest moving all Cassandra related stuff (reIndexing, migration, consistency operation) in the distributed module.

Also, we should annotate this class with @Category(BasicFeature.class) as this is quite critical (and master current behaviour)